src/preprocess.js

```diff
@@ -56,6 +58,210 @@ ${diagramText}
   return JSON.stringify(diagramObject)
 }
 
+// @ts-check
```

Comment:
nitpick: `// @ts-check` is already present line 3, do we need this comment here?

Reply:
done